di_art/lib/error/BadScopeInstanceException.dart

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 DiArt.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
```

Comment:
Can we make the license shorter? E.g. remove last paragraph

Reply:
It's a default short variant, provided by Apache, don't think we need to touch it :)